clippy reports warning in implement_vertex!

This code:
```
#[macro_use]
extern crate glium;

#[derive(Clone, Copy)]
struct MyVtx {
    pos: [f32; 3],
}

implement_vertex!(MyVtx, pos);

fn main() {
}
```

when analyzed with `clippy` reports this warning:
```
warning: calls to `std::mem::forget` with a value that implements Copy. Forgetting a copy leaves the original intact.
 --> src/main.rs:9:1
  |
9 | implement_vertex!(MyVtx, pos);
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: #[warn(forget_copy)] on by default
note: argument has type MyVtx
 --> src/main.rs:9:1
  |
9 | implement_vertex!(MyVtx, pos);
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  = help: for further information visit https://github.com/rust-lang-nursery/rust-clippy/wiki#forget_copy
  = note: this error originates in a macro outside of the current crate
```

Well, `glium::Vertex` does require:
```
error[E0277]: the trait bound `Vtx: std::marker::Copy` is not satisfied
 --> src\main.rs:8:1
  |
8 | implement_vertex!(Vtx, pos);
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `std::marker::Copy` is not implemented for `Vtx`
  |
  = note: required by `glium::Vertex`
```

--- a/src/macros.rs
+++ b/src/macros.rs
@@ -109,13 +109,14 @@ macro_rules! implement_vertex {
                             Cow::Borrowed(stringify!($field_name)),
                             {
                                 // calculate the offset of the struct fields
-                                let dummy: $struct_name = unsafe { ::std::mem::zeroed() };
+                                let dummy: $struct_name = unsafe { ::std::mem::uninitialized() };
                                 let offset: usize = {
                                     let dummy_ref = &dummy;
                                     let field_ref = &dummy.$field_name;
                                     (field_ref as *const _ as usize) - (dummy_ref as *const _ as usize)
                                 };
-                                ::std::mem::forget(dummy);
+                                // NOTE: `glium::vertex::Vertex` requires `$struct_name` to have `Copy` trait
+                                // `Copy` excludes `Drop`, so we don't have to `std::mem::forget(dummy)`
                                 offset
                             },
                             {

